### PR TITLE
Dockerless pause rootfs

### PR DESCRIPTION
**Pull request by cyphar**
This is working on improving the situation described in #91.

### CODE REVIEW

**Comment by mrunalp** [comment]
Nice! This looks fine. I will test this out locally once before merging. Also, we will try to get a Jenkins setup going so we can run all tests on CI.

File: server/sandbox.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	if _, err = os.Stat(podInfraRootfs); err != nil {
 		if os.IsNotExist(err) {
 			// TODO: Replace by rootfs creation API when it is ready
-			if err = utils.CreateFakeRootfs(podInfraRootfs, "docker://kubernetes/pause"); err != nil {
+			if err = utils.CreateInfraRootfs(podInfraRootfs, s.pausePath); err != nil {
```
Comment (runcom) [inline server/sandbox.go]: :)

**Comment by mrunalp** [comment]
LGTM

File: cmd/server/main.go [hunk 2]
```diff
@@ -110,7 +116,9 @@ func main() {
 
 		containerDir := c.String("containerdir")
 		sandboxDir := c.String("sandboxdir")
-		service, err := server.New(c.String("runtime"), c.String("root"), sandboxDir, containerDir, conmonPath)
+		conmonPath := c.String("conmon")
```
Comment (cyphar) [inline cmd/server/main.go]: This actually fixes an issue in #77 with `--conmon` not being set properly.

**Pull request merged**